include/seqan3/alignment/configuration/align_config_min_score.hpp

```diff
@@ -31,13 +31,13 @@ namespace seqan3::align_cfg
  *
  * ### Example
  *
- * \include test/snippet/alignment/configuration/align_cfg_max_error_example.cpp
+ * \include test/snippet/alignment/configuration/align_cfg_min_score_example.cpp
  */
-struct max_error : public pipeable_config_element<max_error, uint32_t>
+struct min_score : public pipeable_config_element<min_score, int32_t>
```

Comment:
Ah ok, this needs to be changed as well. We don't want to use the `.value` from the base anymore. 

```suggestion
struct min_score : public pipeable_config_element<min_score>
```